tests/system/providers/dbt/cloud/example_dbt_cloud.py

```diff
@@ -77,6 +77,12 @@
     )
     # [END howto_operator_dbt_cloud_run_job_sensor]
 
+    # [START howto_operator_dbt_cloud_run_job_sensor_defered]
+    job_run_sensor_defered = DbtCloudJobRunSensor(
+        task_id="job_run_sensor_defered", run_id=trigger_job_run2.output, timeout=20
```

Comment:
```suggestion
        task_id="job_run_sensor_defered", run_id=trigger_job_run2.output, deferrable=True, timeout=20
```
Since this example is showcasing the deferrable functionality, let's add the flag.

Reply:
Thanks for reminding me. Just added it. Thanks!